package.json

```diff
@@ -9,6 +9,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
+    "styled-components": "^6.0.8",
```

Comment:
let's use mui instead of styled-components

Reply:
@NuraiymMamatova  fixed